src/App.tsx

```diff
@@ -1,9 +1,16 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [nameMovies, setSearch] = useState('');
+
+  const visibleMovies = moviesFromServer.filter(list => (
+    list.title.toLowerCase().includes(nameMovies.toLowerCase())
+    || list.description.toLowerCase().includes(nameMovies.toLowerCase())
```

Comment:
Look, you repeat `.toLowerCase().includes(nameMovies.toLowerCase())` not once. Create a method for it